src/promptflow/promptflow/_sdk/operations/_local_storage_operations.py

```diff
@@ -384,12 +392,27 @@ def persist_flow_run(self, run_info: FlowRunInfo) -> None:
         line_run_record.dump(self._run_infos_folder / filename)
 
     def persist_result(self, result: Optional[BulkResult]) -> None:
-        """Persist outputs and metrics from return of executor."""
+        """Persist metrics from return of executor."""
         if result is None:
             return
-        self.dump_outputs(result.outputs)
+        # The executor will persist outputs to output directory, so only dump metrics here for the time being.
         self.dump_metrics(result.metrics)
 
+    def _persist_run_multimedia(self, run_info: Union[FlowRunInfo, NodeRunInfo], folder_path: Path):
+        if run_info.inputs:
+            run_info.inputs = self._serialize_multimedia(run_info.inputs, folder_path)
+        if run_info.output:
+            serialized_output = self._serialize_multimedia(run_info.output, folder_path)
+            run_info.output = serialized_output
+            run_info.result = serialized_output
```

Comment:
I guess we can remove "result" now, it is a legacy concept.

Reply:
Removed.